config/initializers/bundle_dir_roots.yml.erb

```diff
@@ -13,23 +13,23 @@ production:
   - '/thumpers/dpgthumper2-se1/'
   - '/thumpers/sdr'
   - '/arts/'
-  - '/smpl/'
-  - '/smpl2/'
-  - '/smpl3/'
-  - '/smpl4/'
-  - '/smpl5/'
-  - '/smpl6/'
-  - '/smpl7/'
-  - '/smpl8/'
-  - '/smpl9/'
-  - '/smpl10/'
-  - '/smpl11/'
-  - '/smpl12/'
-  - '/smpl13/'
-  - '/smpl14/'
-  - '/smpl15/'
-  - '/smpl16/'
-  - '/smpl17/'
+  - '/media/'
```

Comment:
Let's not rename these mount points in this PR.  They are actually still all called "smplX" on the pre-assembly box, so this would break things if deployed.  If and when they decide to rename the mounts with the help of the Ops team, we can do a new PR to update the mount points as well.

Reply:
Thanks @peetucket!  Wasn't sure about this so put it into it's own commit. Will revert back to previous one and update branch.